test/test_helper.rb

```diff
@@ -149,4 +149,24 @@ def create_and_login_additional_users(num)
       click_button 'Login'
     end
   end
+
+  def admin_order_setup(status)
+    admin = User.create(username: 'admin',
+                        password: 'password',
+                        role:      1)
+
+    ApplicationController.any_instance.stubs(:current_user).returns(admin)
+
+    create_categories_and_items
+    order = Order.create(current_status: status)
+    order_item = OrderItem.new(item_id: Item.last.id, order_id: order.id, quantity: 2)
+    order.order_items << order_item
+    order.save
+
+    visit login_path
+
+    fill_in 'Username', with: 'admin'
+    fill_in 'Password', with: 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.